nvtabular/dask/io.py

```diff
@@ -159,17 +162,28 @@ class DaskDataset:
         to GPU memory capacity). Ignored if part_size is passed
         directly. Note that the underlying engine may allow other
         custom kwargs to override this argument.
+    storage_options: None or dict
+        Further parameters to pass to the bytes backend.
     **kwargs :
         Other arguments to be passed to DatasetEngine.
     """
 
-    def __init__(self, path, engine, part_size=None, part_mem_fraction=0.125, **kwargs):
+    def __init__(
+        self,
+        path,
+        engine=None,
+        part_size=None,
+        part_mem_fraction=None,
+        storage_options=None,
+        **kwargs,
+    ):
 
         if part_size:
             # If a specific partition size is given, use it directly
             part_size = parse_bytes(part_size)
         else:
             # If a fractional partition size is given, calculate part_size
+            part_mem_fraction = part_mem_fraction or 0.125
```

Comment:
Should we just set the default to 0.125 in the method parameters (rather than go part_mem_fraction=None) ?

Reply:
If we set to a default value other than None, we are essentially trying to hide this keyword, right? If so, by setting it to none by default we force the user to understand that this knob exists and inform them they must set it. I feel like that is good behavior training for users because if not, when a user does a groupby operation or a join, if the 0.125 partition is to big they might just think its because the nvtabular doesn't work, and stop using. Or I am just being paranoid here...?

Reply:
So 0.125 is guaranteed to work for all operation sets?

Reply:
There is certainly no guarantee, but this is a good balance between stability and performance in my experience.